pages/index.html

```diff
@@ -0,0 +1,66 @@
+---
+layout: homepage
+title: Homepage
+---
+
+<article>
+  <h2 class="first-grid-item">
+    CBA is an extension for chrome browser automation
+  </h2>
+
+  <img src="/images/Manvel_440x280_border.jpg" alt="CBA-banner" />
+
+  <ol>
+    <li>
+      CBA gives ability to automate chrome browser.
+    </li>
+    <li>
+      Ability to setup step-by-step code injections from Extensions popup
+      interface.
+    </li>
+    <li>
+      Ability to Record interactions in the chrome browser and playing them.
+    </li>
+    <li>
+      Projects saving in browser&#39;s local storage.
+    </li>
+    <li>
+      Step by step code injection.
+    </li>
+    <li>
+      Ability of projects import and export.
+    </li>
+    <li>
+      Ability of using CBA ready functions.
+    </li>
+    <li>
+      Ability to write your own code that will have access to web page&#39;s
+      DOM.
+    </li>
+    <li>
+      And much much more...
+    </li>
+  </ol>
+
+  <section class="box">
+    <h3>Install</h3>
+    <p>
+      <img src="/images/arrow.png" />
+      Get chromium browser automation from google web store.
+    </p>
+    <a
+      href="https://chrome.google.com/webstore/detail/jmbmjnojfkcohdpkpjmeeijckfbebbon"
+      target="_blank"
+      >CBA in chrome web store</a
+    >
```

Comment:
This kind of disorder is forced by prettier. Can we change the corresponding rule?

Reply:
I think it's fine, because href is quite long and in that case it moves each attribute to it's line. Probably there is a way, but having all on 1 line is also a not a good idea.